From cd13c7170a3c73a8cd7a2307386cea401869f0ff Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:19 -0400
Subject: [redhat] kabi: Adding symbol node_states (mm/page_alloc.c)

Message-id: <feeba4eb236d9b89371ee4e297d3c208160bf6b8.1410510773.git.poros@redhat.com>
Patchwork-id: 96128
O-Subject: [PATCH RHEL7.1 139/233] kabi: Adding symbol node_states (mm/page_alloc.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Not referenced directly, node_state gets used due to
    inline function next_online_node which is used to find next numa node.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/node_states
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/node_states
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/node_states

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/node_states b/redhat/kabi/kabi-module/kabi_ppc64/node_states
new file mode 100644
index 0000000..f80f77c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/node_states
@@ -0,0 +1,2 @@
+#1-
+0xb68bfa9d node_states vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/node_states b/redhat/kabi/kabi-module/kabi_s390x/node_states
new file mode 100644
index 0000000..d2b8aae
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/node_states
@@ -0,0 +1,2 @@
+#1-
+0x2bdd6e28 node_states vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/node_states b/redhat/kabi/kabi-module/kabi_x86_64/node_states
new file mode 100644
index 0000000..4aa297c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/node_states
@@ -0,0 +1,2 @@
+#1-
+0xf99d347e node_states vmlinux EXPORT_SYMBOL
-- 
1.7.1